x-pack/test/saml_api_integration/apis/with_basic/saml_login.js

```diff
@@ -0,0 +1,443 @@
+/*
```

Comment:
__note:__ most of this is copy-pasted from the old/existing tests (`../saml_only/saml_login.js`).